tracer/src/Datadog.Trace/ContinuousProfiler/Profiler.cs

```diff
@@ -20,6 +20,7 @@ internal Profiler(IContextTracker contextTracker, IProfilerStatus status)
         public static Profiler Instance
         {
             get { return LazyInitializer.EnsureInitialized(ref _instance, () => Create()); }
+            internal set { _instance = value; }
```

Comment:
This is only `internal` for testing purposes right? Can we document it as such?

Reply:
I extracted it to a separate method (with an explicit name) in https://github.com/DataDog/dd-trace-dotnet/pull/3015/commits/fec54e15782805f7e9565c90f54f5135668fc49d